xblock/VERSION.txt

```diff
@@ -0,0 +1 @@
+0.5.0
```

Comment:
This isn't backwards incompatible, right? No need to go to 0.5.0 over 0.4.13, right?